src/components/Button/Button.tsx

```diff
@@ -1,31 +1,6 @@
 import React from "react";
-import StyledButton from "./Button-style";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { colorType, sizesType, iconType } from "../../types";
-import { icons } from "../../styles/utils";
-
-import { library } from "@fortawesome/fontawesome-svg-core";
-import {
-   faArrowDown,
-   faArrowUp,
-   faArrowRight,
-   faArrowLeft,
-   faPlusCircle,
-   faHeart,
-   faSmile,
-   faTrash,
-} from "@fortawesome/free-solid-svg-icons";
-
-library.add(
-   faArrowUp,
-   faArrowRight,
-   faArrowLeft,
-   faPlusCircle,
-   faArrowDown,
-   faSmile,
-   faHeart,
-   faTrash
-);
+import { SButton, Sicon } from "./style";
+import { colorType, sizesType, iconType } from "../../types/types";
 
 interface Props {

```

Comment:
convention for interfaces - starting with big I and the name so:
IButtonProps could be a good one

Reply:
done